CHANGELOG.md

```
@@ -1,3 +1,12 @@
+<<<<<<< HEAD
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)